modules/standard/HDFS.chpl

```diff
@@ -71,20 +375,20 @@ record hdfsChapelFileSystem {
 
 // --------- Connecting/disconnecting ---------
 
-// Connect to HDFS
+/* Connect to an HDFS filesystem on a single locale */
```

Comment:
Yeah, I don't totally agree with all of the APIs Tim created, but I'm documenting his work rather than re-factoring it at the moment...